frontend/src/interfaces/gem.ts

```diff
@@ -9,3 +12,21 @@ export interface GemColor {
 }
 
 type HexCode = String;
+
+export interface GemInfo {
+  message: string;
+  position: hansel.LatLng;
+  dropper: string;
+  receiver: string;
+  dropTime: Dayjs;
+  color: string;
+}
+
+export enum color {
+  PURPLE = 'purple',
+  PINK = 'pink',
+  BLUE = 'blue',
+  BLACK = 'black',
```

Comment:
feel free ignore: will black gem look unappetizing hahaha

Reply:
> I'm rlly sorry I merged seb's PR without realizing the consequences, you might have to update from master again D:

No worries!!

> feel free ignore: will black gem look unappetizing hahaha
 
I was gonna chope black gem for myself x(

Reply:
<img width="406" alt="Screenshot 2021-09-19 at 12 06 50 AM" src="https://user-images.githubusercontent.com/59130765/133895151-1d64c0f8-7bb5-4510-a0c0-524133853a1a.png">
would you eat

Reply:
omg so spunky I love it. i think it will eat me